command/command.go

```diff
@@ -23,9 +24,15 @@ var version = "v1.0.0-rc.1"
 // empty context
 var nocontext = context.Background()
 
+// PrintAppDetails prints some information about the app
```

Comment:
Consider moving the PrintAppDetails function to a utils package or a separate file instead of keeping it in the main file.